runtime/common/src/lib.rs

```diff
@@ -483,3 +483,56 @@ pub mod xcm_transactor {
 		}
 	}
 }
+
+pub mod foreign_investments {
+	use cfg_primitives::Balance;
+	use cfg_traits::SimpleCurrencyConversion;
+	use cfg_types::{
+		fixed_point::{FixedPointNumberExtension, Rate},
+		tokens::CurrencyId,
+	};
+	use frame_support::pallet_prelude::PhantomData;
+	use sp_runtime::{traits::Get, DispatchError};
+
+	/// Simple stable coin amount converter from one stable to another. Provides
+	/// a synchronous estimation of the amount of one stable currency in
+	/// another, e.g., the worth of 100 EthWrappedDai in USDC.
+	///
+	/// For now, converts any `ForeignAsset` into another with the configured
+	/// rate.
+	///
+	/// NOTE: Should be deprecated ASAP!
+	// TODO(@review): Can we determine whether a ForeignAsset is a stable coin at
+	// this point of time?
```

Comment:
Q: 
> Can we determine whether a ForeignAsset is a stable coin at this point of time?

I don't think so.

Reply:
I think `AssetRegistry::metadata()` will end up with a `pegged` attribute of type `Option<Fiat>` or so that can be used to know if it is a stable coin.

Reply:
cc @offerijns @NunoAlexandre

Reply:
> I think AssetRegistry::metadata() will end up with a pegged attribute of type Option<Fiat> or so that can be used to know if it is a stable coin.

Yes I agree that we will need to add this at some point.

Reply:
@wischli afaik we will only use stable coins as pool currencies? thus `pool_currency: bool` could be used for that purpose?